build/bazel/semver/BUILD

```diff
@@ -1,30 +1,37 @@
-package(default_visibility = ["//visibility:public"])
-
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 load("@rules_proto//proto:defs.bzl", "proto_library")
 
+package(default_visibility = ["//visibility:public"])
+
 licenses(["notice"])
 
 proto_library(
     name = "semver_proto",
     srcs = ["semver.proto"],
 )
 
-alias(
+# Java
+java_proto_library(
     name = "semver_java_proto",
-    actual = "//build/bazel/semver/java:semver_java_proto",
+    deps = ["//build/bazel/semver:semver_proto"],
 )
 
-alias(
+# C++
+cc_proto_library(
     name = "semver_cc_proto",
-    actual = "//build/bazel/semver/cc:semver_cc_proto",
+    deps = ["//build/bazel/semver:semver_proto"],
 )
 
-alias(
+# Go
+go_proto_library(
     name = "semver_go_proto",
-    actual = "//build/bazel/semver/go:semver_go_proto",
+    importpath = "github.com/bazelbuild/remote-apis/build/bazel/semver",
+    proto = ":semver_proto",
 )
 
-alias(
+go_library(
```

Comment:
What's the reason we're not using alias() here? This means that if you end up depending on both `semver_go_proto` and `go_default_library`, you end up getting a bunch of warnings along the lines of:
```
DEBUG: /private/var/tmp/_bazel_ed/9dcffd914bcef91974dd3105239de9ed/external/rules_go~/go/private/actions/link.bzl:59:18: Multiple copies of github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2 passed to the linker. Ignoring bazel-out/darwin_arm64-fastbuild/bin/external/bazel_remote_apis~/build/bazel/remote/execution/v2/remote_execution_go_proto.a in favor of bazel-out/darwin_arm64-fastbuild/bin/external/bazel_remote_apis~/build/bazel/remote/execution/v2/go_default_library.a
```

Reply:
@EdSchouten I don't fully remember. This was the best result I could come up with after spending two days tinkering with this stuff, and trying to make it as clean as possible. There was also a related change for the `remote-apis-sdks` side of things, but I might have forgotten to `git push` that.. 🙈

If you have an idea how to improve upon this, by all means, please go ahead. Happy to review and/or try something out if that helps!

Reply:
No worries; On the Buildbarn side I'm simply adding the following directives to BUILD:
```
# gazelle:resolve go github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2 @bazel_remote_apis//build/bazel/remote/execution/v2:go_default_library
# gazelle:resolve go github.com/bazelbuild/remote-apis/build/bazel/semver @bazel_remote_apis//build/bazel/semver:go_default_library
# gazelle:resolve proto build/bazel/remote/execution/v2/remote_execution.proto @bazel_remote_apis//build/bazel/remote/execution/v2:remote_execution_proto
# gazelle:resolve proto go build/bazel/remote/execution/v2/remote_execution.proto @bazel_remote_apis//build/bazel/remote/execution/v2:remote_execution_go_proto
```
This ensures that both proto rules and plain Go libraries depend on the non-`go_default_library` target.